File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/cafemanagement/castlist/CastListEvent.kt
package com.hhp227.concafe.presentation.main.cafemanagement.castlist

sealed interface CastListEvent {
    data object NavigateBack : CastListEvent
    data class NavigateToCastEdit(val cafeId: String, val castId: String? = null) : CastListEvent
    data class NavigateToSchedule(val castId: String) : CastListEvent
}

File: iosApp/iosApp/Presentation/Main/CafeManagement/CastList/CastListEvent.swift
//
//  CastListEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/07/05.
//

import Foundation

enum CastListEvent {
    case navigateBack
    case navigateToCastEdit(cafeId: String, castId: String?)
    case navigateToSchedule(castId: String)
}
